.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "KaTeX Codespaces",
+  "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:14",
+  "extensions": [
+    "dbaeumer.vscode-eslint",
+    "flowtype.flow-for-vscode",
+    "stylelint.vscode-stylelint",
+    "arcanis.vscode-zipfs"
```

Comment:
What is this used for?

Reply:
@kevinbarabash Yarn 2 uses a zip-based file system, so this extension allows us to browse into a zip file in vscode.